### DISCUSSION TITLE
step0 Scope - Keep the welcome paragraph visible after course start?

### DISCUSSION THREAD

**Discussion by markpatterson27** [skills/.github#24] [Ideas]
Should the welcome paragraph be included in the step0 scope?

The welcome paragraph includes the course details: 'Who is this for', 'What you'll learn', etc. This paragraph is hidden (along with the 'How to start this course' section) when the course is started (i.e. new repo created from template). I think this paragraph is still useful after the course has started and shouldn't be hidden.

Before opening a series of PRs, I was wondering what other peoples opinions are. Should the welcome paragraph remain visible after course start, or should it be hidden out of the way?

**Comment by sinsukehlab** [comment]
I think you can use the template repository as reference.
The 'How to start this course' section tells you to right-click the 'Use this template' button and open the link in a new tab.
Does this mean that you can go back to the tab that shows the template repository?

**Reply by markpatterson27** [reply]
I think it's still useful to have access to the course aims and LO without having to go back to the course template. It could be put in a collapsed `<details>` tag to keep it out of the way after a course is started. Then learners could expand the section if they need a reminder of the course aims, etc.
(I don't think it's necessary to hide it, but others might have a different opinion.)

**Reply by sinsukehlab** [reply]
You have a point. It is better to put it in a collapsed `<details>` tag.

**Reply by sinsukehlab** [reply]
I have opened a PR from this discussion: skills/template-template#20
